app/decorators/nav/link_builder.rb

```diff
@@ -0,0 +1,279 @@
+# frozen_string_literal: true
+
+# Defines nav menus and related helper methods.
+module Nav
+  class LinkBuilder < ApplicationDecorator
+    delegate_all
+
+    attr_accessor :context
+
+    def initialize
+      self.context = {}
+    end
+
+    def nav_items
+      sample_period = Work::Period.new(community: h.current_community)
+      sample_job = Work::Job.new(period: sample_period)
+      sample_shift = Work::Shift.new(job: sample_job)
+      items = [
+        {
+          name: :people,
+          path: lens_path_if_present("users", index_path: h.users_path),
+          permitted: h.policy(User).index?,
+          icon: "users"
+        }, {
+          name: :meals,
+          path: lens_path_if_present("meals/meals", index_path: h.meals_path),
+          permitted: h.policy(Meals::Meal.new(community: h.current_community)).index?,
+          icon: "cutlery"
+        }, {
+          name: :work,
+          path: lens_path_if_present("work/shifts", index_path: h.work_shifts_path),
+          permitted: h.policy(sample_shift).index_wrapper?,
+          icon: "wrench"
+        }, {
+          name: :reservations,
+          path: lens_path_if_present("reservations", index_path: h.reservations_path),
+          permitted: h.policy(Reservations::Reservation.new(resource:
+                                                            Reservations::Resource.new(community: h.current_community))).index?,
```

Comment:
Can we please move this code above so we're creating a `sample_resource` and then `sample_reservation`? This looks kind of gnarly.

Reply:
I'm not sure what you mean here.  Do you mean that we add, above those lines, code to create sample_resource etc and then use those sample_ s in the permitted: line?

Reply:
Yes, just like it's being done with the other sample objs. Optional though, if you'd rather just be done with this!

Reply:
That's done.